src/ViewComposers/CountryLocaleViewComposer.php

```diff
@@ -3,8 +3,13 @@
 namespace PodPoint\I18n\ViewComposers;
 
 use Illuminate\View\View;
+use League\ISO3166\ISO3166;
+use PodPoint\I18n\CountryCode;
 use Illuminate\Config\Repository;
 
+/**
+ * @property array $countryLocalesOptions
```

Comment:
I added it just so we can get the IDE to auto-complete nicely from the consumer (like `dealerships-tool` for example)

```php
<?php
/** @var \PodPoint\I18n\ViewComposers\CountryLocaleViewComposer $view */
$view->countryLocalesOptions;
```

Reply:
Looking at it again, I think it's kinda not working because `$view` is not just an instance of `CountryLocaleViewComposer` but also can have more ViewComposers loaded in...

Will remove it...